### PR TITLE
Change functions to es6 arrow functions

### LINKED ISSUE #170: Convert es5 to es6 code
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Script.js contains es5code
**Expected behavior:**
Replace es5 code to es6 wherever applicable. For example replace ordinary functions with arrow functions. 

**Would you like to work on the issue?**
No.

### PR DESCRIPTION
**Pull request by rupav**
Fixes #170 one task.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Converted all the function in es5 to es6 fat arrow functions, in all js files.
Solves the issues, first part.

@zamhaq @ms10398 @gabru-md  let me know if this is how you guys wanted.

### CODE REVIEW

**Comment by rupav** [comment]
@gabru-md please review now, resolved the conflicts. cc: @zamhaq @stealthanthrax

File: src/script.js [hunk 1]
```diff
@@ -203,15 +201,15 @@ function composeReplyTable(response, columns, data) {
     response.tableType = true;
     response.table = table;
     return response;
-}
+};
 
-function composeSusiMessage(response, t, rating) {
+let composeSusiMessage = (response, t, rating) => {
```
[the remark this answers is not in the document]
Comment (rupav) [reply src/script.js]: Already changed limits in PR https://github.com/fossasia/susi_chromebot/pull/387/files

[the remark this answers is not in the document]
Comment (ms10398) [reply src/script.js]: Can you rebase it?

**Comment by rupav** [comment]
Agreed @ms10398, that why I have written this PR solves 1 task of the issue. Since I need to keep in mind the hoisting of variables, I will convert var to let and const appropriately in the next and final PR for the same issue

**Comment by ms10398** [comment]
Sure you can and check if anything can be destructured

Sometime in near future I am looking at the extension using Babel and Gulp to build it like this https://github.com/EmailThis/extension-boilerplate

or maybe port it to React

but sadly I don't have time for it right now :worried:

**Comment by rupav** [comment]
@gabru-md @zamhaq already made the changes for max statements to 60 in my previous PR.
Refer https://github.com/fossasia/susi_chromebot/pull/387#issuecomment-426213556

File: src/script.js [hunk 2]
```diff
@@ -201,9 +199,9 @@ function composeReplyTable(response, columns, data) {
...
```
[the remark this answers is not in the document]
Comment (rupav) [reply src/script.js]: same issue in PR #387 but solved by changing maxstatements value in that PR already.

**Comment by gabru-md** [comment]
we can iterate over it now. thanks

**Pull request merged**